Wikipedia/Code/HintController.swift

```diff
@@ -0,0 +1,196 @@
+@objc(WMFHintPresenting)
+protocol HintPresenting: AnyObject {
+    var willDragCompletion: (() -> Void)? { get set }
```

Comment:
This is a bit unclear - could it be a ref to the `HintController` and on drag explicitly call `dismissHint` or new method `dismissHintDueToUserInteraction` if it needs to be differentiated?

Reply:
sounds good 👍